README.md

```diff
@@ -298,6 +298,7 @@ More detail on Elm at Microsoft in
   ([GitHub](https://github.com/BeautifulDestinations)) - Beautiful
   Destinations is the award-winning creative agency behind the largest travel
   community on social media.
+* [Emirates](https://www.emirates.com/) - National airline of the United Arab Emirates.
```

Comment:
🎉 Well done to the team and kudos to the person who convinced management to "OK" this! 🥇😉